packages/import/src/ImportCsv.ts

```diff
@@ -49,14 +53,28 @@ export class ImportCsv {
                 .makeOptionMandatory()
             )
             .addOption(new Option("--dir <dirPath>", "The directory used as input for files"))
+            .addOption(new Option("--batch-size <batchSize>", "The size of one batch (i.e. numbers of items in one extrinsic")
+                .default(10)
+                .argParser(parseInt)
+            )
+            .addOption(new Option("--local", "Connect to local node (--env value ignored)")
```

Comment:
I am not sure this should be reserved to the sole developers, we might have users which would like to test their procedure locally with `logion-test` in the same way as we do.

Reply:
I agree, and I will unhide this option. Note that, to be fully working, it should be possible to configure the `logion_classification` and `CC4.0` LOC id's.